### PR TITLE
integrated OPEN AI for subjective evaluations

### PR DESCRIPTION
**Pull request by anipnwr7777**
Intergrated OPEN_AI to evaluate subjective answers against a prompt and return a score and a rationale.

### CODE REVIEW

**Review by yash0212 — CHANGES_REQUESTED** [review]
Please resolve the added comments

File: services/code.service.js [hunk 1]
```diff
@@ -107,6 +113,51 @@ const _prepareErrorMessage = (outputLog, language, command) => {
     return errorMsg.trim()
 }
 
+
+const executePrompt = async (req, res, response) => {
```
[the remark this answers is not in the document]
Comment (anipnwr7777) [reply services/code.service.js:117]: done

File: services/code.service.js [hunk 2]
```diff
@@ -124,63 +175,71 @@ const execute = async (req, res) => {
...
-            compileLog.error !== undefined ? _prepareErrorMessage(compileLog, language, compileCommand) : ''
-
-        // Check if there is no compilation error
-        if (response.compileMessage === '') {
-            let command
-            if (language === 'java') {
-                // Remove ulimit as a temp fix
-                command = `cd /tmp/ && timeout ${langConfig.timeout}s ${langConfig.run}`
+    if (req.language === 'promptv1') {
+        if (req.language === 'promptv1') {
```
[the remark this answers is not in the document]
Comment (anipnwr7777) [reply services/code.service.js:182]: sorry for the typo, removed it.

**Review by yash0212 — APPROVED** [review]
LGTM

**Pull request closed without merging**